src/main/java/task/Task.java

```diff
@@ -0,0 +1,68 @@
+package task;
+
+/**
+ * Interface for list items in Babe,
+ */
+public abstract class Task {
+    /**
+     * String icon for marked.
+     */
+    static String MARKED = "[X]";
```

Comment:
Consider using a common prefix for the static Strings?

Reply:
That is a good point! Thank you!